inc/functions_search.php

```diff
@@ -213,11 +213,18 @@ function clean_keywords($keywords)
 	if(my_strpos($keywords, "or") === 0)
 	{
 		$keywords = substr_replace($keywords, "", 0, 2);
+		$keywords = " ".$keywords;
 	}
 
 	if(my_strpos($keywords, "and") === 0)
 	{
 		$keywords = substr_replace($keywords, "", 0, 3);
+		$keywords = " ".$keywords;
+	}
+
+	if(!$keywords)
```

Comment:
I think this check would be better in the `perform_search_mysql` function, based on the following usages:

- [`privatemessage_perform_search_mysql`](https://github.com/mybb/mybb/blob/f538e4403bd76036af5f6f0dc4be86c84563df60/inc/functions_search.php#L492) - already checks if there are keywords and does the same error.
- [`helpdocument_perform_search_mysql`](https://github.com/mybb/mybb/blob/f538e4403bd76036af5f6f0dc4be86c84563df60/inc/functions_search.php#L809) - already checks if there are keywords and does the same error.

Otherwise, this looks good.

Reply:
@euantorano - is possible to finish this without @rajat315315 if he will not respond? Quite useful fix 👍